src/main/webapp/i18n/en/exam.json

```
@@ -680,7 +680,8 @@
         },
         "examTimeline": {
             "title": "Exam timeline of {{ username }}",
-            "breadcrumb": "Exam timeline"
+            "breadcrumb": "Exam timeline",
+            "disabledTooltip": "You cannot view the exam timeline as long as the student exam has not been submitted."
```

Is this warning meant to explain why it is not enabled during the conduction or also after the exam when the student did not submit their exam? Depending on that, this (or something similar) might fit better:
```suggestion
            "disabledTooltip": "You cannot view the exam timeline if the student exam has not been submitted."
```